### PR TITLE
new header logo and updated navigation

### PR DESCRIPTION
**Pull request by LucaFumis**

### CODE REVIEW

**Comment by opentargets-fern** [comment]
Deploy preview ready!

Built with commit e9680cd5777cef51c3c2016f82559ca266824d39

https://deploy-preview-2--journalist-crab-36716.netlify.com

File: _config.yml [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
-email: support@targetvalidation.org
+email: contact@opentargets.org
 description: > # this means to ignore newlines until "baseurl:"
   Open Targets is a public-private initiative to generate evidence on the validity of therapeutic targets based on genome-scale experiments and analysis. 
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "" # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: targetvalidate
 github_username:  opentargets
-
+linkedin: https://www.linkedin.com/company/centre-for-therapeutic-target-validation
```
Comment (elipapa) [inline _config.yml]: i moved all the repeated values inside the config.yml so that it's easier to keep consistent across pages in the future.

**Comment by elipapa** [comment]
i moved all external URL to parameters in the `_config.yml`. This will be helpful if we ever switch to a new theme.

@LucaFumis i fixed the hamburger (locally i could see it, but when deployed the checkbox looks invisible again)

**Comment by elipapa** [comment]
Looks great! And fast!!!
My only 3 comments: 
1) footer image don't fold well in mobile view 
image

2) Projects pages have **way** too much text. Particularly in mobile...there is lots of scrolling to do. Could a solution be collapsing the text + a "view more ..." button/link ? Something using [bootstrap js collapse](http://getbootstrap.com/javascript/#collapse) perhaps?

3) People page also requires a bunch of scrolling. and the text folds on the side of the person, making it difficult to read.
image
Now that we have bootstrap, should we make the bio fold under the images and make it collapsable as well (like point 2)?

**Comment by LucaFumis** [comment]
Sorry Eliseo I had missed your comments.

*Projects* - content length was also mentioned by David, and I understand it will be re-worked by Holly / David
*Layout* issues - yup, that wrapping is not ideal. I'm updating the various parts to a layout more similar to the webapp, but it wasn't a straightforward plug'n' play change

**Comment by LucaFumis** [comment]
>i moved all external URL to parameters in the _config.yml. This will be helpful if we ever switch to a new theme.
Ah fantastic - I'll update the relevant bits accordingly

**Comment by LucaFumis** [comment]
These are the changes / feedback discussed with David last week:

1. *homepage*: update text of blue box
2. *footer*: add "terms of use" etc. as per current website
3. *Overview*: link partners to respective websites 
4. *Projects*: link to platform
5. *People*: add info for Will Chen. Also update info for Denise and CK as discussed with them. *Platform team*: rearranged content of lines due to spacing issues
6. *Navigation*: change order of links (these had to be hardcoded)
7. *Blog* : Denise confirmed there is no need for a "news" section in the navigation; discussed possibility of having drop-down menus later on.

For later on:
*Blog*: it would be good to have a link back to the corporate site
*Publications*: might be added in the future

**Pull request merged**